src/Core/Services/IApplicationCacheService.cs

```diff
@@ -10,6 +10,7 @@ public interface IApplicationCacheService
     Task<IDictionary<Guid, OrganizationAbility>> GetOrganizationAbilitiesAsync();
 #nullable enable
     Task<OrganizationAbility?> GetOrganizationAbilityAsync(Guid orgId);
+    Task<IDictionary<Guid, OrganizationAbility?>> GetManyOrganizationAbilityAsync(IEnumerable<Guid> orgIds);
```

Comment:
I'm not sure we need this - you're basically just recreating a subset of the dict returned by `GetOrganizationAbilitiesAsync`. You could just call that method instead and look up the same orgAbilities from the result.

Reply:
Removed!